interpreter/mesh/src/main/scala/io/buoyant/interpreter/MeshInterpreterInitializer.scala

```diff
@@ -43,7 +43,7 @@ case class Retry(
 case class MeshInterpreterConfig(
   dst: Option[Path],
   root: Option[Path],
-  tls: Option[TlsClientConfig],
+  tls: Option[ClientTlsConfig],
```

Comment:
AFAICT this was just a placeholder, as `io.l5d.mesh` wasn't supported on the server?